Makefile.PL

```diff
@@ -64,6 +64,7 @@ my %requires = (
   'Email::Sender'                       => 0,
   'FFI::Platypus'                       => 0,
   'Future'                              => '0.34',
+  'Graph'                               => 0,
```

Comment:
What do the number values represent here?

Reply:
in Makefile.pl 0, means any version will do meaning the latest (if installing from scratch) or whatever is currently installed if not.  If an actual number is defined, it must be that version or higher. So if you run it on a system with a version less than, it will generate an error and stop.